apps/builddao/widget/events/Calendar.jsx

```diff
@@ -66,39 +134,74 @@ const handleMonthChange = (change) => {
 const Toolbar = () => {
   return (
     <StyledToolbar>
-      <div className="section gap-3">
-        <span>{dateString}</span>
-        <button className="date-changer" onClick={() => handleMonthChange(-1)}>
-          <i className="bi bi-chevron-left"></i>
-        </button>
-        <button className="date-changer" onClick={() => handleMonthChange(1)}>
-          <i className="bi bi-chevron-right"></i>
-        </button>
-      </div>
-      <div className="section gap-1 justify-content-center">
-        <Button
-          variant={selectedView === "month" ? "secondary" : "outline"}
-          onClick={() => setSelectedView("month")}
-        >
-          Month
-        </Button>
-        <Button
-          variant={selectedView === "list" ? "secondary" : "outline"}
-          onClick={() => setSelectedView("list")}
-        >
-          List
-        </Button>
-      </div>
-      <div className="section justify-content-end" style={{ gap: 10 }}>
-        <Button variant="outline" onClick={() => setShowFilterModal(true)}>
-          Filter By
-        </Button>
-        {context.accountId && (
-          <Button variant="primary" onClick={() => setShowCreateModal(true)}>
-            Add Event
+      <HeaderLeft>
+        <div className="section gap-3">
+          <span>{dateString}</span>
+          <button
+            className="date-changer"
+            onClick={() => handleMonthChange(-1)}
+          >
+            <i className="bi bi-chevron-left"></i>
+          </button>
+          <button className="date-changer" onClick={() => handleMonthChange(1)}>
+            <i className="bi bi-chevron-right"></i>
+          </button>
+          {context.accountId && (
+            <button
+              className="date-changer"
+              style={{
+                border: `1px solid ${theme === "light" ? "#cdcdcd" : "rgba(255, 255, 255, 0.2)"}`,
+                padding: "4px 12px",
+                widht: "32px",
+                height: "32px",
+                borderRadius: "8px",
+                display: "flex",
+                alignItems: "center",
+                justifyContent: "center",
+              }}
+              onClick={() =>
+                Storage.set("event_theme", theme === "light" ? "dark" : "light")
+              }
+            >
+              <i className={theme === "light" ? "bi bi-moon" : "bi bi-sun"}></i>
```

Comment:
Add a prop with ability to show or hide this button so that on gateways with global theme we can set it using prop